### ISSUE TITLE
When adding a new address for a user required fields don't highlight

### ISSUE THREAD

**Issue by Virgil3** [itsyouonline/identityserver#659]
When a user tries to authorize an account the address field must be filled in. The problem is in the address form that pops up if you don't fill in all fields (including label) clicking ADD will not do anything. The remaining unfilled mandatory fields are not being highlighted.

![iyo address field](https://user-images.githubusercontent.com/31761294/31888210-c93adffe-b803-11e7-9a40-f243f8c06b57.png)

Users are getting stuck on this step because they get no feedback from clicking ADD. We've gotten reports of people getting stuck for more than 20 minutes on this or even people giving up.

**Comment by LeeSmet** [comment]
Moved to bugfix release since KYC milestone is no longer relevant

**Comment by LeeSmet** [comment]
Rather then highlighting, lets disable the button if not all required fields are filled in. Although this draws less attention to the particular fields, it is a better solution for mobile, where users might need to scroll down to see the fields, thus still remaining stuck in the endless loop if the button is clickeable

**Comment by Virgil3** [comment]
People will get confused by the disabled button. It sounds trivial but it will happen. For mobile users we could force scroll their page to the unfilled field when they click authorize.

**Comment by LeeSmet** [comment]
I have considered leaving the button clickeable and rendering errors on the required fields, while also focusing the first invalid field. This would indeed be the best solution. Unfortunately, angularjs provides no easy way to focus inputs. Therefore, this has been implemented as is as it should unblock the majority of people. Note that simply rendering errors would not help as in some mobile configurations it is not clear that the user needs to scroll down apparently, hence the reason the focus is also required for this approach.
